test/e2e/spark_dependencies_test.go

```diff
@@ -42,12 +43,12 @@ func sparkTest(t *testing.T, f *framework.Framework, testCtx *framework.TestCtx,
 	}
 	defer undeployJaegerInstance(j)
 
-	err = WaitForCronJob(t, f.KubeClient, namespace, fmt.Sprintf("%s-spark-dependencies", name), retryInterval, timeout)
+	err = WaitForCronJob(t, f.KubeClient, namespace, fmt.Sprintf("%s-spark-dependencies", name), retryInterval, timeout+timeout+1*time.Minute)
```

Comment:
Can you change it to something easier to read? Like, `5 * time.Minute` (or whatever the calculation yields)

Reply:
@jpkrohling I would but then @objectiser might object.  He suggested I do it this way for this PR https://github.com/jaegertracing/jaeger-operator/pull/847  :smirk:

Reply:
I thought it best to be relative to the global `timeout` value - however in this PR it looks like `timeout` has been added two times? Not sure this is necessary.